ui/appui-react/src/appui-react/toolbar/GroupButtonItem.tsx

```diff
@@ -468,8 +468,8 @@ export class ToolbarGroupItem extends React.Component<ToolbarGroupItemComponentP
 /** @internal */
 const getFirstItem = (groupItem: GroupButton): ActionButton | GroupButton | undefined => {
   for (const item of groupItem.items) {
-    const isHidden = ConditionalBooleanValue.getValue(item.isHidden);
-    const isDisabled = ConditionalBooleanValue.getValue(item.isDisabled);
+    const isHidden = ConditionalBooleanValue.getValue(item.isHidden); // eslint-disable-line deprecation/deprecation
```

Comment:
`Conditional*Value` are not deprecated (marked in many places, will not raise others)

Reply:
Good catch, initially I thought we could get away with moving conditionals into `core-react`. Removed these left-over deprecations.